gnomad_qc/v3/resources/release.py

```diff
@@ -63,6 +63,92 @@ def release_ht_path(
         return f"gs://gnomad/release/{release_version}/ht/gnomad.{data_type}.r{release_version}.sites.ht"
 
 
+def release_sites(public: bool = False) -> VersionedTableResource:
+    """
+    Retrieve versioned resource for sites-only release Table.
+
+    :param public: Determines whether release sites Table is read from public or private bucket. Defaults to private
+    :return: Sites-only release Table
+    """
+    return VersionedTableResource(
+        default_version=CURRENT_RELEASE,
+        versions={
+            release: TableResource(
+                path=release_ht_path(release_version=release, public=public)
+            )
+            for release in RELEASES
+        },
+    )
+
+
+def release_header_path(
+    release_version: str = CURRENT_RELEASE, hgdp_tgp_subset: bool = False
+) -> str:
+    """
+    Fetch path to pickle file containing VCF header dictionary.
+
+    :param release_version: Release version. Defaults to CURRENT RELEASE
+    :param hgdp_tgp_subset: Whether to return the header for the HGDP + 1KG subset. Default will return the header
+        path for the full release.
+    :return: Filepath for header dictionary pickle
+    """
+    subset = ""
+    if hgdp_tgp_subset:
+        if release_version not in HGDP_TGP_RELEASES:
+            raise DataException(
+                f"{release_version} is not one of the available releases for the HGP + 1KG subset: {HGDP_TGP_RELEASES}"
+            )
+        subset = f"_hgdp_tgp"
+
+    return f"gs://gnomad/release/{release_version}/vcf/genomes/gnomad.genomes.v{release_version}_header_dict{subset}.pickle"
+
+
+def release_vcf_path(
+    release_version: str = CURRENT_RELEASE,
+    hgdp_tgp_subset: bool = False,
+    contig: Optional[str] = None,
+) -> str:
+    """
+    Fetch bucket for release (sites-only) VCFs.
+
+    :param release_version: Release version. Defaults to CURRENT RELEASE
+    :param hgdp_tgp_subset: Whether to get path for HGDP + 1KG VCF. Defaults to the full callset (metrics on all samples)        sites VCF path
+    :param contig: String containing the name of the desired reference contig. Defaults to the full (all contigs) sites VCF path
+        sites VCF path
+    :return: Filepath for the desired VCF
+    """
+    if hgdp_tgp_subset:
```

Comment:
should this function also have this check? `if release_version not in HGDP_TGP_RELEASES:`